ui/raidboss/data/07-dt/raid/r4s.ts

```diff
@@ -537,11 +720,21 @@ const triggerSet: TriggerSet<Data> = {
         data.witchHuntAoESafe = aoeOrder[0];
 
         // assumes Near first; if Far first, just reverse
-        let baitOrder: NearFar[] = ['near', 'far', 'near', 'far'];
-        if (data.witchHuntBait === undefined)
+        let baitOrder: (NearFar | RoleBait)[];
+
+        if (data.witchHuntBait === 'near') {
+          if (data.triggerSetConfig.witchHunt === 'DN')
+            baitOrder = ['tank', 'healer', 'melee', 'ranged'];
+          else
+            baitOrder = ['near', 'far', 'near', 'far'];
+        } else if (data.witchHuntBait === 'far') {
+          if (data.triggerSetConfig.witchHunt === 'DN')
+            baitOrder = ['far', 'near', 'far', 'near'];
+          else
+            baitOrder = ['healer', 'tank', 'ranged', 'melee'];
```

Comment:
```suggestion
          if (data.triggerSetConfig.witchHunt === 'DN')
            baitOrder = ['healer', 'tank', 'ranged', 'melee'];
          else
            baitOrder = ['far', 'near', 'far', 'near'];
```

Reply:
Oh good call!

Reply:
This should be fixed now thanks for spotting it!